sdk/core/azure-core/src/main/java/com/azure/core/http/ContentType.java

```diff
@@ -4,7 +4,17 @@
 package com.azure.core.http;
 
 /**
- * The different values that commonly used for Content-Type header.
+ * <p>This class provides constants for commonly used Content-Type header values in HTTP requests and responses.</p>
+ *
+ * <p>It includes constants for the following Content-Type header values:</p>
+ * <ul>
+ *     <li>{@link #APPLICATION_JSON}: Represents a JSON Content-Type header.</li>
+ *     <li>{@link #APPLICATION_OCTET_STREAM}: Represents a binary Content-Type header.</li>
+ *     <li>{@link #APPLICATION_X_WWW_FORM_URLENCODED}: Represents a form data Content-Type header.</li>
+ * </ul>
```

Comment:
Principle question: is it necessary to list the members of the enum in this javadoc? IDE completion or javadoc will list the members and the members have good comments.